### ISSUE TITLE
Lightpaper crash; where is the (new & unsaved) document I was editing?

### ISSUE THREAD

**Issue by apgarmd** [42Squares/LightPaper#64]
Hello - 

I hope you can help.

Lightpaper recently crashed on me [El Capitan for reasons that remain unclear ([here is the crash log](http://pastebin.com/raw/Xkj5GCcm)). 

Upon restart my document was gone.  It was a brand new document that I had been editing for the past hour or so and I had not yet named or saved it manually -- however, in settings I had already configured Autosave 'every two minutes'  --yet I am unable to find where Lightpaper keeps documents that are being 'Autosaved'. Help?  

【Mac Pro; 10.11.4, 3.5Ghz 6-core xeon e5】

Thank you for a wonderful program --that is getting even better in important ways.

**Comment by ashokgelal** [comment]
Unfortunately, there is no way to recover a temporary/ unsaved file. Right now autosave only kicks in if you have already saved the file once.

**Comment by apgarmd** [comment]
Thank you for the reply.

tldr;

   - Respectfully, imho then the (lack of) a high-class Autosave is a
   curiously unbecoming departure and miss for an app such as Lightpaper; The
   impact that its short-comings can create as-is are among the most serious
   down-side experience any App can render; Hence the position I feel must be
   taken is LP autosave is woefully insufficient; its actual behavior and
   benefits rendered are (unintentionally of course) dangerously misleading;
   --and as a sideways complement;  I would suggest its something like a
   step-function away from the consistent strength and enjoyable, efficient
   and powerful ux that its become...


Tbh it has been a long time since I have been on the receiving end of this
veritable cliche ---i.e. "remember to save your work" ⋯

A) You have my deep appreciation and respect for what Lightpaper,
specifically in how well you have evolved and consistently stayed true to
what must be a fairly religious set of principles in terms of staying
focused and not adding every feature requested...

I feel its fair to say that "auto-save" and inherent mechanism to help
prevent data loss (at the application later) is a relatively high stakes
area --if you don't do a great job of it that doesn't mean necessarily a
change in the value of the app --*provided the interface and "can't miss
it" messaging make it clear where those risks may lie *(non-common/general).

I'm mostly over this particular incident now, but I want to be honest and
say that it was difficult to accept (esp. given my modest though
non-trivial experience with a spectrum of Apps/Servers/Networking on *nix
(&& osx that spans nearly 20 years. Even the *fear even a remote potential
for that kind of punch to the gut moment: "Poof, it's all gone!" *(and
subsequent sense that its 90%+ user error) seems like a Win95/98 relic

Anyway, on a happier and less pedantic note —LightPaper is obviously going
places and a lot of thought has to have gone into =everything= in order to
product a top-self app as it is. I will remain a fan and active user
indefinitely
no doubt...

Meaning, (and it being just some guy talkin') my feedback and general
notions were not intended to turn into a lecture, yet at the same time I
decided to reply without pulling too many punches as I don't want to
short-change you from receiving a context and color around this issue I
opened.

Just one last sentiment:  I *believe* that its become imminently reasonable
to say especially involving a feature like Autosave 【esp. given the
accompanying button copy so strongly suggesting the App *"Automatically
save every two minutes"*】that as-is, some might say its borderline
irresponsible. I have to believe almost anyone would see that and conclude
summarily their "mindless and carefree protection is built-in" --but maybe
I'm wrong.

It seems an easy thing to at least significantly mitigate --but I'll leave
that much at least up to the expert. Thanks for reading,


M
